commands/commands.go

```diff
@@ -0,0 +1,53 @@
+package commands
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Commands mapping of top level commands
+var Commands map[string]*Command
+
+// SetCommands set all command mappings
+func SetCommands() {
```

Comment:
This is really nice; good job!

Code style suggestion:  break each command into its own function... `setPingCommand`, `setSayUpperCommand` etc.

Reply:
Thanks!

Agreed separating to function would be best.

The assigned "action" functions can be stored in there individual files too.